pytrip/models/proton.py

```diff
@@ -108,20 +107,25 @@ def rbe_mcnamara(dose, let, abx):
     """
 
     _apx = 0.999064 + 0.35605 * let / abx
-    _bpx = 1.1012 - 0.0038703 * np.sqrt(abx) * let
-    _bpx *= _bpx
+    _sbpx = 1.1012 - 0.0038703 * np.sqrt(abx) * let
 
-    rbe = _rbe_apx(dose, _apx, _bpx, abx)
+    rbe = _rbe_apx(dose, _apx, _sbpx, abx)
     return rbe
 
 
-def _rbe_apx(dose, apx, bpx, abx):
+def _rbe_apx(dose, apx, sbpx, abx):
     """
     :params dose: proton dose      [Gy]
     :params apx: alpha_p / alpha_x [dimensionless] RBE_max = ap/ax when (dose -> 0 Gy)
-    :params bpx: beta_p / beta_x   [dimensionless] RBE_min = sqrt(bp/bx) when (dose -> inf Gy)
+    :params sbpx: beta_p / beta_x  [dimensionless] RBE_min = sqrt(bp/bx) when (dose -> inf Gy)
     :params abx: alpha_x / beta_x  [Gy^-1]
     """
 
-    _rbe = (np.sqrt(abx*abx + 4*abx*apx*dose + 4*bpx * dose*dose) - abx) / (2 * dose)
+    _rbe = 1.0 / (2.0 * dose)
+    if hasattr(_rbe, '__iter__'):
+        _rbe[_rbe == np.inf] = 0
```

Comment:
Following case, assuming we have a region where RBE=0 (due to model limitation), we have some reference radiation dose D_phys = X, and calculate biological dose as 
D_phys / RBE. We obtain `np.inf`.

In this case if RBE would be np.inf, then D_biol would be 0.

Giving user an option (with default setting of RBE=0) would allow them more flexibility in this case.